docs/guide/built-in-functions.md

```diff
@@ -218,6 +218,7 @@ Total number of functions: **{{ $page.functionsCount }}**
 | COLUMNS     | Returns the number of columns in the given reference.                                                               | COLUMNS(Array)                                      |
 | FORMULATEXT | Returns a formula in a given cell as a string.                                                                      | FORMULATEXT(Reference)                              |
 | HLOOKUP     | Searches horizontally with reference to adjacent cells to the bottom.                                               | HLOOKUP(Search_Criterion, Array, Index, Sort_Order) |
+| HYPERLINK   | Creates a hyperlink inside a cell.                                                                                  | HYPERLINK(Url[, LinkLabel])                         |
```

Comment:
```suggestion
| HYPERLINK   | Stores the url in the cell's metadata. It can be read using method [`getCellHyperlink`](../api/classes/hyperformula.md#getcellhyperlink) | HYPERLINK(Url[, LinkLabel])                         |
```

Reply:
doc updated